src/vm/finalizerthread.cpp

```diff
@@ -505,7 +502,7 @@ VOID FinalizerThread::FinalizerThreadWorker(void *args)
 
     BOOL bPriorityBoosted = FALSE;
 
-    while (!fQuitFinalizer)
+    for (;;)
```

Comment:
How is this loop exited? Exception? longjmp?

Reply:
I have deleted too much here. I need to add some of this back.